package.json

```diff
@@ -52,6 +52,7 @@
         "dagre": "~0.8.5",
         "dagre-d3": "~0.6.4",
         "flatgeobuf": "3.25.0",
+        "geoengine-openapi-client": "file:/home/beilschmidt/git/geo-engine/openapi-client/typescript/dist",
         "immutable": "~4.3.0",
```

Comment:
I get this warning when I run it:

> Warning: /home/michael/git/geoengine-ui/dist/core/fesm2022/geoengine-core.mjs depends on 'geoengine-openapi-client'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Reply:
We needed to add `/esm` to the end.
The current package works fine: [43cddc1](https://github.com/geo-engine/geoengine-ui/pull/687/commits/43cddc17438a148c50093ad2371ff3f3485dbf1f)